easy_access/services/merge.py

```
@@ -0,0 +1,301 @@
+        new_item: Dictionary with new field values
+        db_item: Existing database item (Tortoise model instance)
+        fielddict: Dictionary of field names to ordering rules
+        changes: Dictionary to record changes in
+
+    Returns:
+        Tuple of (changes dict, updated db_item)
+    """
+    if not changes:
+        changes = {
+            "material_id": new_item.get("material_id"),
+            "update_time": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
```

Ruff (0.14.7)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)